R/retrieve_credentials.R

```diff
@@ -0,0 +1,93 @@
+#' View available credentials keys in the system key-value store
+#'
+#' @export
+view_credential_keys <- function() {
+
+  get_keyring_creds_table() %>%
+    dplyr::select(-key_name) %>%
+    dplyr::rename(keyname = service)
+}
+
+#' Retrieve metadata and authentication values from an on-disk credentials file
+#'
+#' @noRd
+get_smtp_file_creds <- function(file_name = NULL) {
+
+  # For the given `file_name`, read in the JSON
+  # data and convert it into a list object
+  readLines(file_name) %>%
```

Comment:
readLines/writeLines calls should use UTF-8

Reply:
Now added.